### PR TITLE
added duel contracts and example

### PR DESCRIPTION
**Pull request by xxlongzaixx**

### CODE REVIEW

File: dfktools/duel/duel.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
+from web3 import Web3
+
+CONTRACT_ADDRESS = '0xE97196f4011dc9DA0829dd8E151EcFc0f37EE3c7'
```
Comment (NichJohn83) [inline dfktools/duel/duel.py:4]: should probably move this to its own constants file

**Comment by NichJohn83** [comment]
Are all these functions just to get duel information? Or is there a way to actually send heroes on duels?

**Comment by big-analytics** [comment]
Also looking forward for sending heroes!

File: dfktools/duel/duel.py (hunk 1 above)
Comment (xxlongzaixx) [reply dfktools/duel/duel.py:4]: I follow the project structure where the contract address is stored in each respectively folder python file

**Comment by xxlongzaixx** [comment]
> Also looking forward for sending heroes!



> Are all these functions just to get duel information? Or is there a way to actually send heroes on duels?

the latest commit has the feature to send heroes on duels for ranked and private match

**Pull request merged**